netty/src/main/java/io/grpc/netty/GrpcSslContexts.java

```diff
@@ -268,6 +281,12 @@ private static Provider findJdkProvider() {
             || JettyTlsUtil.isJava9AlpnAvailable()) {
           return provider;
         }
+      } else if (IBM_PROVIDER_NAME.equals(provider.getName())) {
+        if (JettyTlsUtil.isJettyAlpnConfigured()
+            || JettyTlsUtil.isJettyNpnConfigured()
+            || JettyTlsUtil.isJava9AlpnAvailable()) {
```

Comment:
lines 285-287 are identical to lines 279-281. The checks seem to be independent of `provider` (the for loop iterator) and loop invariant. Can we create a helper method and also initialize the value outside the loop?